include/multi_blas_helper.cuh

```diff
@@ -125,23 +125,34 @@ namespace quda
     */
     constexpr int max_N_multi_1d() { return 24; }
 
+    /**
+       @brief Return the maximum size supported by multi-blas kernels
+       (max_N_multi_1d()) rounded down to the largest power of two.
+       This is used for the NXZ power-of-two instantiation.
+    */
+    constexpr int max_N_multi_1d_pow2()
+    {
```

Comment:
What's the reason for going up to `v >> 16` and not, say `v >> 32`?

Reply:
v >> 32 doesn't tell you much with a 32-bit number 😉

Reply:
...yep, fair. Need more coffee.